flow360/component/simulation/simulation_params.py

```diff
@@ -490,6 +490,41 @@ def _update_entity_private_attrs(self, registry: EntityRegistry) -> EntityRegist
 
         return registry
 
+    def _get_model_by_type(self, model_type: ModelTypes) -> ModelTypes | List[ModelTypes]:
+        """
+        Given the model type, returns the model in the simulation params
+
+        Returns
+        -------
+        The single model or a list of model of the model_type
+        """
+        if not self.models:
+            return None
+
+        models = []
+        # pylint: disable=not-an-iterable
+        for model in self.models:
+            if isinstance(model, model_type):
+                models.append(model)
+
+        if len(models) == 0:
+            return None
+        if len(models) == 1:
+            return models[0]
+        return models
+
+    @property
+    def fluid(self) -> Fluid:
+        """
+        Returns the fluid model of the simulation params
+
+        Returns
+        -------
+        Fluid
+            The fluid model in the params
+        """
+        return self._get_model_by_type(model_type=Fluid)
```

Comment:
although I understand desire to make it accessible from top level, I think it will create more confusion, why all the sudden fluid is a top level and other models are not? On contrary if we put more things on top level, we will lose hierarchical structure and come back to flat structure. Note that user might want to do `params.fluid = fl.Fluid(...)`, such assignment is not allowed.

How about we keep it under models and do:
```python
params.models.fluid
```
We need to extend "List" type. This way we can also do: `fluid`, `solid` `rotations`, `bet_disks` etc...